_includes/scenarios/integrate/establish-a-relationship-to-another-identity.md

```diff
@@ -0,0 +1,164 @@
+
+```jsonc
+{
+  "id": "<ID of RelationshipTemplate>",
+  "secretKey": "<secret key of RelationshipTemplate>"
+}
+```
+
+When the RelationshipTemplate of the Templator is successfully loaded onto the Connector, a success response is sent. This looks like the success response you receive when you have [successfully created a RelationshipTemplate]({% link _docs_integrate/establish-a-relationship-to-another-identity.md %}#successfully-created-relationshiptemplate) on the Templator, except that the value of the property `isOwn` is now `false` instead of `true`. Assuming that there is no Relationship between the Connector and the Templator yet and that the [RelationshipTemplate]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplate) contains a [RelationshipTemplateContent]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplatecontent) in its `content` property, the Connector will additionally receive a new incoming Request. The Integrator of the Connector can accept it if they want to [send a Relationship Request]({% link _docs_integrate/establish-a-relationship-to-another-identity.md %}#send-it-as-a-connector) to the Templator.
+
+### Make it available to an App user
+
+If an App user wants to send a Relationship Request to the Templator, the App user must first scan a QR Code that contains the reference to a RelationshipTemplate which is owned by the Templator. To create this QR Code on the Templator, proceed as described in the documentation of the [Get RelationshipTemplate]({% link _docs_use-cases/use-case-transport-get-relationship-template.md %}) use case, use the `id` of the [created RelationshipTemplate]({% link _docs_integrate/establish-a-relationship-to-another-identity.md %}#successfully-created-relationshiptemplate) and specify the value `image/png` in the `Accept` header field. After scanning the QR Code, the App user receives the conditions for establishing a Relationship to the Templator as specified in the RelationshipTemplate. If these are accepted, the App user can now [send a Relationship Request]({% link _docs_integrate/establish-a-relationship-to-another-identity.md %}#send-it-as-an-app-user) to the Templator.
+
+## Send a Relationship Request
+
+You have learned so far how to create a RelationshipTemplate on the Templator and make it available to other Identities. How another Identity, the so-called Requestor, can use it to send a Relationship Request and finally establish a Relationship to the Templator is explained in the next steps.
```

Comment:
I don't think this recap is necessary here. I guess it's an artifact from putting the two pages together. Equally, you don't need to foreshadow the section following this one.